doc/funding/process.md

```diff
@@ -0,0 +1,10 @@
+# Process for funding
+
+Our process for funding is still a working draft.
+This is the process the board currently follows, and it will improve the details of this process as we begin taking our first steps.
```

Comment:
```suggestion
This is the process the Board currently follows, and it will improve the details of this process as we begin taking our first steps.
```